### ISSUE TITLE
Serve all db files in a folder

### ISSUE THREAD

**Issue by stratosgear** [simonw/datasette#1380]
I tried to get the `serve` command to serve all the .db files in the `/mnt` folder but is seems that the server does not refresh the list of files.

In more detail:

* Starting datasette as a docker container with: 
```
docker run -p 8001:8001 -v `pwd`:/mnt \
    datasetteproject/datasette \
    datasette -p 8001 -h 0.0.0.0 /mnt
```

* Datasette correctly serves all the *.db files found in the /mnt folder 
* When the server is running, if I copy a new file in the $PWD folder, Datasette does not seem to see the new files, forcing me to restart Docker.

Is there an option/setting that I overlooked, or is this something missing?

BTW, the `--reload` setting, although at first glance is what you think you need, does not seem to do anything in regards of seeing all *.db files.

Thanks!

**Comment by Segerberg** [comment]
There is no such option see https://github.com/simonw/datasette/issues/43

**Comment by glasnt** [comment]
Would it make sense to run datasette with a fswatch/inotifywait on a folder, then?

**Comment by Segerberg** [comment]
@glasnt yeah I guess that could be an option.  I run datasette on large databases > 75gb and the startup time is a bit slow for me even with -i --inspect-file options. Here's a quick sketch for a plugin that will reload db's in a folder that you set for the plugin in metadata.json. If you request /-reload-db new db's will be added. (You probably want to implement some authentication for this =) ) 

`from datasette.database import Database
from datasette import hookimpl
from datasette.utils.asgi import Response
import os


async def reload_db(datasette):
    spec = datasette.plugin_config('datasette-reload-db')
    db_dir = os.listdir(spec['dir'])
    databases = datasette.databases
    for db in databases:
        if db == '_internal':
            continue
        datasette.remove_database(db)

    for db in db_dir:
        datasette.add_database(Database(
            datasette,
            path=os.path.join(spec['dir'], db),
            is_mutable=True
        ))
    return Response.redirect('/')


@hookimpl
def register_routes():
    return [
        (r"^/-reload-db", reload_db)
    ]


@hookimpl
def menu_links(datasette):
    return [
        {"href": datasette.urls.path("/-reload-db"), "label": "Reload Databases"},
    ]
`
